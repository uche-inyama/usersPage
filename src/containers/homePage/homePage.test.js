import React from 'react';
import { Provider } from 'react-redux';
import { shallow } from 'enzyme';
import configureStore from 'redux-mock-store';
import HomePage from './homePage';

const mockStore = configureStore();
const store = mockStore();

const props = {
  centers: [{
    building: 'Albion',
    hall: 'Tulip',
  }],
};

describe('Appointment as a connected component', () => {
  it('', () => {
    const wrapper = shallow(
      <Provider store={store}>
        <HomePage
          building={props.centers.building}
          hall={props.centers.hall}
        />
      </Provider>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
