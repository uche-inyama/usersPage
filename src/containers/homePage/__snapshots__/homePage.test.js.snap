// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Appointment as a connected component  1`] = `ShallowWrapper {}`;
